src/ajax/parseXML.js

```diff
@@ -9,7 +9,8 @@ jQuery.parseXML = function( data ) {
 		return null;
 	}
 
-	// Support: IE9
+	// Support: IE 9-11 only
```

Comment:
I'm not adamant, but I agree with @dmethvin that it'd be nice not to be so strict that we need to go back and edit all Support comments.